op-devstack/sysgo/l2_proposer_validity.go

```diff
@@ -78,6 +101,54 @@ func WithValidityConfigDirsOption(
 	))
 }
 
+func WithVPSubmissionInterval(n uint64) ValidityProposerOption {
+	return ValidityProposerOption(func(p devtest.P, id stack.L2ProposerID, cfg *ValidityProposerConfig) {
+		cfg.submissionInterval = &n
+	})
+}
+
+func WithVPRangeProofInterval(n uint64) ValidityProposerOption {
+	return ValidityProposerOption(func(p devtest.P, id stack.L2ProposerID, cfg *ValidityProposerConfig) {
+		cfg.rangeProofInterval = &n
+	})
+}
+
+func WithVPRangeProofEvmGasLimit(n uint64) ValidityProposerOption {
+	return ValidityProposerOption(func(p devtest.P, id stack.L2ProposerID, cfg *ValidityProposerConfig) {
+		cfg.rangeProofEvmGasLimit = &n
+	})
+}
+
+func WithVPMaxConcurrentProofRequests(n uint64) ValidityProposerOption {
```

Comment:
```
if n == 0 {
    p.Require().Fail("maxConcurrentProofRequests must be > 0")
}
```

nit: Should we consider adding validations like ^? And also add a safe guard on the proposer side as well?